### ISSUE TITLE
Sources for names

### ISSUE THREAD

**Issue by fgregg** [datamade/probablepeople#1]
http://www.cs.cmu.edu/~einat/datasets.html

**Comment by martinburch** [comment]
Here's a big list of drug and medical device manufacturer corporation names:
https://openpaymentsdata.cms.gov/dataset/AMGPO-Lookup-File/2j4a-fwnz

After @derekeder 's NICAR-L announcement, I ran all the names starting with "A" through the bulk parser and found a 6% error rate. (Companies tagged as people.) One of the bigger contributors to the error rate was single tokens tagged as people. For example, AXOGEN was tagged as a person but 
AXOGEN CORPORATION was tagged as a corporation. If we are evaluating only western names, I think single tokens have a high probability of being corporation names.

**Comment by cathydeng** [comment]
looks like 85/1148 were parsed incorrectly, & most failures were more than one token, but had some tricky words that the parser hasn't seen before. these failures are great training examples, thanks @martinburch! I'll work on adding some of these to the training data.

**Comment by Downchuck** [comment]
Surname counts from 2000 US Census
http://www.ssa.gov/oact/babynames/decades/century.html

Given name counts from Social Security, born 1800 - present.
http://www.census.gov/topics/population/genealogy/data/2000_surnames.html

**Comment by cathydeng** [comment]
hey @Downchuck - we already have the most common names from the census. do any of these datasets contain full name strings, instead of name components that are already split out?

**Comment by Downchuck** [comment]
No, far as I understand they're already split up; the death index has pairs (full names); I don't know that they'd be of use though.
